src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/IssueFinalDecisionSelectRecipients.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.sptribs.caseworker.event.page;
+
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.callback.AboutToStartOrSubmitResponse;
+import uk.gov.hmcts.sptribs.ciccase.model.*;
+import uk.gov.hmcts.sptribs.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.sptribs.common.ccd.PageBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class IssueFinalDecisionSelectRecipients implements CcdPageConfiguration {
+
+    @Override
+    public void addTo(PageBuilder pageBuilder) {
+        pageBuilder
+            .page("issueFinalDecisionSelectRecipients", this::midEvent)
+            .pageLabel("Select recipients")
+            .label("LabelRecipients","Who should receive this decision notice?")
+            .complex(CaseData::getCicCase)
+            .optional(CicCase::getRecipientSubjectCIC, "")
+            .optional(CicCase::getRecipientRepresentativeCIC,"")
+            .optional(CicCase::getRecipientRespondentCIC,"")
+            .done();
+    }
+
+    public AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseData, State> details,
+                                                                   CaseDetails<CaseData, State> detailsBefore) {
+        final CaseData data = details.getData();
+        final List<String> errors = new ArrayList<>();
+
+        if (null != data.getCicCase()
```

Comment:
Could we use isEmpty() is similar instead of null checks and size = 0 checks? or if not move to a separate validation method to clean up this area?

Reply:
Replaced using Spring CollectionUtils.